packet_packer.go

```diff
@@ -116,7 +114,11 @@ func (p *packetPacker) packPacket(stopWaitingFrame *frames.StopWaitingFrame, lea
 
 	var payloadFrames []frames.Frame
 	if isHandshakeRetransmission {
-		payloadFrames = append(payloadFrames, stopWaitingFrame)
+		// Find the SWF
+		if p.stopWaiting == nil {
+			return nil, errors.New("PacketPacker BUG: Handshake retransmissions must contain a StopWaitingFrame")
+		}
+		payloadFrames = append(payloadFrames, p.stopWaiting)
 		// don't retransmit Acks and StopWaitings
 		for _, f := range handshakePacketToRetransmit.Frames {
```

Comment:
Oh, you changed that in a later commit. I was reviewing this PR commit by commit, so I just saw that now.